app/src/androidTest/java/org/mozilla/fenix/ui/TabbedBrowsingTest.kt

```diff
@@ -257,7 +257,6 @@ class TabbedBrowsingTest {
 
     @Test
     fun verifyTabTrayNotShowingStateHalfExpanded() {
-        homeScreen { }.dismissOnboarding()
```

Comment:
Not sure about this change...that would imply that the workflow has changed? but if it run ~50 times successfully let's give it a try this way...

Reply:
Yes, the nav bar is visible now without dismissing the onboarding. The problem failing this test was actually in this step, so since we don't need it, it can just be removed.